### DISCUSSION TITLE
Cloning Communities?

### DISCUSSION THREAD

**Discussion by borko-sla** [knowledgefutures/pubpub#1282] [Support] [answered]
Hi all, just a quick question about whether or not it's possible (now or in the future) to clone communities. I'm working with a university instructor who is using PubPub as a structured collaboration platform for their students, but hopes to 'reset' the community after the semester. I appreciate any insights!

**Comment by gabestein** [comment, marked as answer]
Hello! Yup, we have some internal tooling that can clone communities. Send a note to hello@pubpub.org with the community url and what you're looking for, and we should be able to help.
